cellprofiler/gui/help/content.py

```diff
@@ -1522,9 +1522,11 @@ def __image_resource(filename):
 In this case, the procedure to set up the input modules to handle these
 files is as follows:
 
--  In the **Images** module, drag-and-drop your folders of images into
-   the File list panel. If necessary, set your rules accordingly in
-   order to filter out any files that are not part of a movie sequence.
+-  In the **Images** module, drag-and-drop your folders of images into the
+   File list panel. If necessary, set your rules accordingly in order to
+   filter out any files that are not part of a movie sequence. By default,
+   only image files with Bio-Formats extensions are included, which
```

Comment:
It might be nice to add a link: https://docs.openmicroscopy.org/bio-formats/5.7.1/supported-formats.html